packaging/ubuntu-trusty/debian/source/format

```diff
@@ -0,0 +1 @@
+3.0 (quilt)
```

Comment:
Consider whether `3.0 (quilt)` is needed if we're not actually patching the source. Could this be `3.0 (native)` or event `1.0`?